main/settings.py

```diff
@@ -716,6 +716,16 @@
 CELERY_RESULT_SERIALIZER = "json"
 CELERY_ACCEPT_CONTENT = ["json"]
 CELERY_TIMEZONE = "UTC"
+CRON_COURSERUN_SYNC_HOURS = get_string(
+    name="CRON_COURSERUN_SYNC_HOURS",
+    default=0,
+    description="'hours' value for the 'sync-courseruns-data' scheduled task (defaults to midnight)",
+)
+CRON_COURSERUN_SYNC_DAYS = get_string(
+    name="CRON_COURSERUN_SYNC_DAYS",
+    default=None,
```

Comment:
```suggestion
    default="*",
```

If we're defaulting this to run once every day, why not make `"*"` the actual default value for the setting? If you do this, you can replace `CRON_COURSERUN_SYNC_DAYS or "*"` with just `CRON_COURSERUN_SYNC_DAYS`